pkg/domain/stateDiffDogu.go

```diff
@@ -160,13 +166,10 @@ func getActionsForPresentDoguDiffs(expected DoguDiffState, actual DoguDiffState)
 	return neededActions
 }
 
-func appendActionForMinVolumeSize(actions []Action, expectedSize *ecosystem.VolumeSize, actualSize *ecosystem.VolumeSize) []Action {
-	if expectedSize != nil && actualSize != nil {
-		if expectedSize.Cmp(*actualSize) == 1 {
-			return append(actions, ActionUpdateDoguResourceMinVolumeSize)
-		}
+func appendActionForMinVolumeSize(actions []Action, expectedSize ecosystem.VolumeSize, actualSize ecosystem.VolumeSize) []Action {
```

Comment:
Is this okay? Dogus without the minVolmeSize in the Blueprint will always trigger a "update volume size" action.